Bottom: 83ec14097473851b1f72bfa6e643d79b714718cc
Top:    943ea0efe7fedbad8482e2cd9c2be9939758e4cf
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-27 16:42:57 -0700

Refresh of lldb-python-fix-another-fflush

---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 5a75187262a..5d025a46dce 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -845,6 +845,7 @@ class FileHandleTestCase(lldbtest.TestBase):
             def i(sbf):
                 for i in range(10):
                     f = sbf.GetFile()
+                    self.assertEqual(f.mode, "w")
                     yield f
                     sbf = lldb.SBFile.Create(f, borrow=True)
                     yield sbf
